imod/mf6/out/__init__.py

```diff
@@ -217,7 +221,11 @@ def open_hds_like(
 
 
 def open_cbc(
-    cbc_path: FilePath, grb_path: FilePath, flowja: bool = False
+    cbc_path: FilePath,
+    grb_path: FilePath,
+    flowja: bool = False,
+    simulation_start_time: Optional[np.datetime64] = None,
```

Comment:
simulation_start_time is a moment in time. time_unit is a time unit. Those two together do not form an interval definition, or any physical concept afaik. So I'm a bit reluctant to group them together, as there are just 2. If there come more, we can group them together as some fabrication class ("time_information_needed_for_output_conversion" or something like that).

Reply:
I think we have to avoid burdening the user too much with having to instantiate iMOD Python specific instances. ``Package`` objects etc. are fine, but separate objects for specific arguments is getting a bit much. Our users are used to using ``pandas``, where these two have to be provided separately.